src/lib/object_store/SessionObject.cpp

```diff
@@ -217,7 +217,8 @@ bool SessionObject::deleteAttribute(CK_ATTRIBUTE_TYPE type)
 }
 
 // The validity state of the object
-bool SessionObject::isValid()
+// the doRefresh parameter has no meaning for this implementation since noting is stored on disk.
+bool SessionObject::isValid(const bool doRefresh __attribute__((unused)))
```

Comment:
The build fails for windows with
`SessionObject.cpp(221,50): error C3646: '__attribute__': unknown override specifier`

You might need to replace `const bool doRefresh __attribute__((unused))` with something like `const bool /* doRefresh */`, or adding `(void)doRefresh;` in the function body.

Reply:
how stupid of me. I did just assume the the windows build was not ready yet.
Thanks @bjosv